docs/Usage.md

```diff
@@ -91,6 +91,7 @@ GATEWAY: "10.10.10.1"                                         # The gateway for
 IP_PREFIX: "25"                                               # Subnet Mask in CIDR notation for your node IP ranges
 DNS_SERVERS: "[8.8.8.8,8.8.4.4]"                              # The dns nameservers for the machines network-config.
 BRIDGE: "vmbr1"                                               # The network bridge device for Proxmox VE VMs
+VLAN: "2"                                                     # (optional) The VLAN to put the bridge device into
```

Comment:
```suggestion
VLAN: "2"                                                     # The VLAN to put the bridge device into
```

Reply:
if you want, you can also set the default value in the template. so it's considered optional.

Reply:
I think setting it mandatory would break the previous flow as existing configs will complain. Need to have a look which default works here. From `*uint16` it should be `null`… 🤔

Reply:
Updated the templates to have a `null` default value.

Reply:
the null will not work for most cases, 
and you don't need to add VLAN to all cluster templates.

Reply:
Why will the `null` not work? The default of an unset pointer is `null` and the type is a `*uint16`… 🤔